app/models/trust_level3_requirements.rb

```diff
@@ -143,11 +143,7 @@ def min_days_visited
   end
 
   def num_topics_replied_to
-    @user.posts
-      .public_posts
-      .where("posts.created_at > ? AND posts.post_number > 1", time_period.days.ago)
-      .select("distinct topic_id")
-      .count
+    @user.user_stat.calc_topic_reply_count!(min_topics_replied_to * 2, time_period.days.ago)
```

Comment:
Why `*2` and not just `+1`?

Reply:
TL3 requirements checked for `.public_posts` https://github.com/discourse/discourse/pull/9769/files#diff-71b275fc6f97d344de4978707d0f2fdeL147